content/en/continuous_integration/intelligent_test_runner/how_it_works.md

```diff
@@ -0,0 +1,61 @@
+---
+title: How It Works
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/"
+    tag: "Blog"
+    text: "Streamline CI testing with Datadog Intelligent Test Runner"
+  - link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
+    tag: "Blog"
+    text: "Monitor all your CI pipelines with Datadog"
+---
+
+## What is Intelligent Test Runner?
+
+Intelligent Test Runner is Datadog's test impact analysis solution. Test impact analysis is a technique that has been gaining popularity over the past few decades. However, it's typically hard and time-consuming to implement. Intelligent Test Runner simplifies this complexity with an easy setup process.
+
+Test impact analysis maps each test to the set of code files in your repository that your test uses. Its goal is to skip tests that are not affected by the code changes that are being introduced, in order to reduce the time taken by the testing phase of your CI.
```

Comment:
re: warning
change last sentence to, 
"Its goal is to skip tests that are not affected by the code changes that are being introduced. This leads to a direct reduction in time spent testing in CI.